profiles/Dijkstra-setup.yaml

```diff
@@ -308,43 +380,276 @@ hand_armor: *hand_armor # check the gear list to see whats going on here
 
 crossing_training:
 - Locksmithing
+- Trading
 - Athletics
 - Outdoorsmanship
-# - Forging
-# - Outfitting
+- Forging
+- Outfitting
 - Attunement
 - Utility
 - Augmentation
 - Warding
-- Trading
 - Mechanical Lore
 
 appraisal_training:
 - pouches
 - gear
 
+full_pouch_container: backpack
+
+crafting_training_spells:
+  Augmentation:
+    abbrev: FIN
+    name: Finesse
+    mana: 5
+    symbiosis: true
+  Utility:
+    abbrev: NOU
+    mana: 5
+    symbiosis: true
+  Warding:
+    abbrev: NON
+    mana: 5
+    symbiosis: true
+
 listen: true
 listen_observe: true
 
 sell_loot_pouch: false
 spare_gem_pouch_container: duffel bag
 gem_pouch_adjective: purple
-sell_pouches_for_trading: yes # same as true
-
-# safe_room_empath:
-# safe_room_id:
+sell_pouches_for_trading: true
+sale_pouches_container: backpack
 
 repair_timer: 86400 # Repair once every 24 hours
 repair_every: .inf # Infinity - Dont use repair counter
 
+# To correctly stow special types of 'stones' you need to subtract the general 'stones' from your lootables list. Keep in mind that you now need to list all types of stones to loot in loot additions, e.g. 'shmurgle stones' wont be looted anymore with the settings below
 loot_subtractions:
 - stones
 loot_additions:
 - kyanite stones
 - jadeite stones
 - waermodi stones
+- lantholite stones
 loot_specials:
 - name: jadeite stones
   bag: duffel bag
 - name: kyanite stones
   bag: duffel bag
+
+restock_shop_inside_room: 14024
```

Comment:
💡 What do you think about having `restock_shop` be a 'complex setting', so something like:
```yaml
restock_shop:
    inside_room: 14024
    outside_room: 7913
    ...
```

Reply:
I like that idea, ill make the change

Reply:
👍